src/database/pipelines/populateAgendaItems.ts

```diff
@@ -0,0 +1,27 @@
+import { fetchAgendaItems } from '@/api/agendaItem';
+import { insertAgendaItems } from '@/database/queries/agendaItems';
+
+export const populateAgendaItems = async (start: Date, end: Date) => {
+  let insertedCount = BigInt(0);
+  while (start < end) {
```

Comment:
Why do we need to go through each month one at a time if the internal helper is already doing one page at a time?

Reply:
i'm not sure any of the usual reasons i try to avoid doing massive queries by pagination apply here, but some general reasons:
- the page number is a happenstance property of the data which could change at any moment, whereas a static property like the date ought to be more consistent. e.g. if i run the script once today, and it fails halfway thru, and i want to run it again without re-running it over the old data, i have no guarantee the page numbers will still line up properly, but the dates should still be good
- depending on how the database is set up and the exact query situation, asking for the full data set and going thru it one page at a time could trigger a full data scan on each query in a way that asking for a specific slice of data might not